packages/lesswrong/server/dialogues/cron.ts

```diff
@@ -25,6 +26,30 @@ addCronJob({
   }
 });
 
+
+addCronJob({
```

Comment:
Actually, if you're going to handle this via callbacks going forward, but just want a one-time backfill, you don't want to use a cron job for that, just a `manualMigration`.

Reply:
I say we could, not that we necessarily would. For now it's built this way, so I'd say let's just ship and maybe one glorious day it'll be callbacked?